enhancements/sig-policy/66-delete-policy-resources/README.md

```diff
@@ -0,0 +1,237 @@
+# Removing Objects That Are Created by Policies Upon Policy Deletion
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+When the `remediationAction` of a policy is set to enforce and an object is referenced
+in one of its `ConfigurationPolicy` objects that does not exist, the config policy
+controller will create that object on the managed cluster. However, when the policy
+is deleted, there is no logic to remove that object, so it will remain on the cluster
+indefinitely until another policy is created to clean it up (this is how the majority
+of our E2E tests handle clean-up) or the object is manually deleted via the CLI. We 
+propose keeping track of the objects that are created by each `ConfigurationPolicy` in the status of the
+`ConfigurationPolicy` and adding a new field to the `ConfigurationPolicy` spec that allows
+users to specify whether they want to delete any objects created by the `ConfigurationPolicy`.
+
+## Motivation
+
+The current implementation, which leaves old objects on the cluster, can make it a
+pain to remove policies and confusing for users if they delete a policy without
+keeping track of the objects that policy created. There is no way for users to
+know which objects are left over after a policy is deleted, so those objects
+will persist on the cluster and have potential to cause confusion. Even if users
+know which objects need to be cleaned up after deleting a policy, it is time-consuming
+to do so. By allowing users to specify that they would like to automatically delete
+managed objects, we would make policy clean-up significantly simpler and easier.
+
+### Goals
+
+1. Object deletion: objects that are created by policies will be deleted when
+the policy is deleted
+2. Opt-in: add an optional new field to the `ConfigurationPolicy` spec and default
+to leaving the object on the cluster upon policy deletion so that the behavior
+of old policies is not changed
+
+### Non-Goals
+
+1. Deleting objects not created by the parent policy: we will not support
+deleting objects that are not directly created by a `ConfigurationPolicy` when
+that `ConfigurationPolicy` is deleted (i.e. existing resources that are edited
+by the `ConfigurationPolicy`). We determined implementing this would create
+more problems than it would solve, such as users accidentally deleting objects
+that are created by the cluster or other policies.
+
+## Proposal
+
+We would add a new field to the `ConfigurationPolicy` spec called
```

Comment:
I would mention that this only applies when the `remediationAction` of `enforce` is set.